noetic/distribution.yaml

```diff
@@ -933,6 +933,19 @@ repositories:
       url: https://github.com/ros/convex_decomposition.git
       version: melodic-devel
     status: unmaintained
+  core_perception:
+    doc:
+      type: git
+      url: https://github.com/Autoware-AI/core_perception.git
```

Comment:
It looks like [the last melodic release](https://github.com/ros/rosdistro/pull/26284) uses `nobleo` here instead of `Autoware-AI`. Which is correct?

Reply:
Ah, we didn't do any of the documentation. Only the release (as per https://github.com/Autoware-AI/core_perception/issues/7).
But it's better to be consistent I guess. I'll change it to our fork.

Reply:
Ah oops, you're right there's no `doc` section in the last release. Thanks for making it consistent!